samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -229,3 +229,61 @@ HorizontalSquaredVirtualGridList.storyName = 'Horizontal Squared VirtualGridList
 HorizontalSquaredVirtualGridList.parameters = {
 	propTables: [Config]
 };
+
+class SnapToCenterVGL extends Component {
+	componentDidMount () {
+		this.scrollTo({index: 1, animate: false, focus: true, stickTo: 'center'});
+	}
+
+	renderItem = ({index, ...rest}) => {
+		const {source} = items[index];
+		let style;
+		if (index === 0 || index === items.length - 1) {
+			style = {
+				visibility: 'hidden'
+			};
+		}
+
+		return (
+			<ImageItem {...rest} src={source} style={style} />
+		);
+	};
+
+	getScrollTo = (scrollTo) => {
+		this.scrollTo = scrollTo;
+	};
+
+	render () {
+		return (
+			<VirtualGridList
+				cbScrollTo={this.getScrollTo}
+				dataSize={updateDataSize(number('dataSize', Config, 10))}
+				itemRenderer={this.renderItem}
+				itemSize={{
+					minWidth: ri.scale(number('minWidth', Config, 1300)),
```

Comment:
When I change the `minWidth` and `datasize` on knob. 500 and 50 respectively.
Item 0 and Item 49 was not shown

Reply:
Since the requirement should show the first item and the last item in the center, I added style to be hidden for Item 0 and Item 49.